submissions/TaniaBondarenko/a-tiny-js-world-update/index.js

```diff
@@ -0,0 +1,75 @@
+class Inhabitant {
+  constructor(name, gender, saying, friends,legs, hands) {
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+    this.legs = legs;
+    this.hands = hands;
+  }
+  toString() {
+    return (` I\'m <b>${this.name}</b>, a ${this.gender}. To my friend ${this.friends} I always say: "${this.saying}"`);
```

Comment:
No good reasons for this class to let its children handle shared properties, not handling those by itself.

Reply:
Understood. Fixed and updated.